app/src/main/res/layout/activity_loginform.xml

```diff
@@ -7,68 +7,82 @@
     android:background="@drawable/background"
     tools:context=".login.LoginFormActivity">
 
-    <Button
-        android:id="@+id/loginButton"
-        android:layout_width="110dp"
-        android:layout_height="35dp"
-        android:layout_marginTop="36dp"
-        android:background="@drawable/btn_rounded"
+    <TextView
+        android:id="@+id/greetingText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         android:fontFamily="@font/bold"
-        android:onClick="loginBtn_OnClick"
-        android:text="Login"
-        android:textColor="@color/blue"
+        android:text="Welcome to"
+        android:textColor="@color/green"
+        android:textSize="25sp"
+        app:layout_constraintBottom_toBottomOf="@+id/offline_button"
         app:layout_constraintEnd_toStartOf="@+id/guideline"
-        app:layout_constraintHorizontal_bias="0.071"
+        app:layout_constraintHorizontal_bias="0.37"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/passwordlog"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.09" />
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="117dp"
```

Comment:
Take care, on some phone with smaller screen size, fixed values can become problematic

Reply:
thanks I am aware of that :). I will try a bit more to have hard coded values.